go/libkb/paperkey_phrase.go

```diff
@@ -34,7 +34,8 @@ func MakePaperKeyPhrase(version uint8) (PaperKeyPhrase, error) {
 
 // NewPaperKeyPhrase converts a string into a PaperKeyPhrase.
 func NewPaperKeyPhrase(phrase string) PaperKeyPhrase {
-	return PaperKeyPhrase(strings.ToLower(phrase))
+	s := strings.Join(strings.Fields(phrase), " ")
```

Comment:
Sorry I forgot to address it earlier.  I changed the order, but for now I left `type PaperKeyPhrase string` unchanged.  

Besides being a bigger change to move to []string components, the full string with the spaces is the phrase, so seems to me to be the right thing to store in memory.  But if you feel strongly, let me know.